src/main/java/structures/Arrayer.java

```diff
@@ -21,9 +29,14 @@ public static int[] evensOnly(int[] input) {
      * 
      * ex: [1004, 9181, 1700, 4565] -> [4, 1, 0, 5]
      */
+
+    // establishing a for loop to evalute the input and determine the remainder which is the last digits
```

Comment:
-1 style, comments should go directly *above* or next to the code they describe.